README.md

```diff
@@ -105,6 +105,26 @@ the next 100 are measured for inference throughput.
 The input parameters to the test are used to generate permutations
 of tests with varying configurations.
 
+### The model
+As Neural Network, we use Pytorch's implementation of Resnet50. The script `imagenet/model_saver.py`
+can be used to generate the model for CPU or GPU. 'By navigating to the imagenet folder, it can be invoked with
+
+```bash
```

Comment:
Its a little confusing with having the command before the explanation, I would say -> to generate for CPU use this command... to generate with GPU use this command... instead of having it after

Reply:
Done!